pom.xml

```diff
@@ -159,7 +159,7 @@
         <mongo.java.driver.version>3.6.4</mongo.java.driver.version>
         <mockito.version>2.10.0</mockito.version>
         <spark.xml.version>0.5.0</spark.xml.version>
-        <scopt.version>3.7.0</scopt.version>
+        <scopt.version>4.0.0-RC2</scopt.version>
```

Comment:
btw. are there any of the v4+ features actually being used?

Reply:
Yes, indeed. The feature for combining parsers